src/test/store/__snapshots__/profile-view.test.js.snap

```diff
@@ -494,6 +494,8 @@ Object {
           "F",
           "Load 6: https://mozilla.org",
           "Load 7: https://mozilla.org",
+          "Bailout",
+          "Invalidate",
```

Comment:
This change and the one below are due to the fact that because we now return from `setup` the result of the selector `getMarkerGetter`, we exercize `extractMarkerDataFromName`. In this function, we mutably change the string table: https://github.com/firefox-devtools/profiler/blob/f21340395a171dfd0847fb8dab077f57405ab430/src/profile-logic/marker-data.js#L167-L168

Maybe that's a problem, maybe not, but anyway this is out of scope for this PR :)